grpl

```diff
@@ -2,8 +2,6 @@
 set -e
 export GRPL_WORKDIR=$(cd $(dirname $0) && pwd)
 export FULL_COMMAND="$0 $@"
-# shellcheck source=common
-. "$GRPL_WORKDIR/common"
```

Comment:
removed this explicit common import in grpl because

- logs re-direction and suppression commands run at the end of the script, so if we import commons here, we'll see logs related to re-requsite tools installation, etc
- common is already being imported in cluster/install and example/deploy file so we can skip importing it here